google-cloud-pubsub/test/helper.rb

```diff
@@ -204,6 +206,33 @@ def subscription_hash topic_name, sub_name,
     hsh
   end
 
+  def create_subscription_args sub_name,
+                               topic_name,
+                               push_config: nil,
+                               ack_deadline_seconds: nil,
+                               retain_acked_messages: false,
+                               message_retention_duration: nil,
+                               labels: nil,
+                               enable_message_ordering: nil,
+                               filter: nil,
+                               dead_letter_policy: nil,
+                               retry_policy: nil
+    [
+      subscription_path(sub_name),
+      topic_path(topic_name),
+      push_config: push_config,
+      ack_deadline_seconds: ack_deadline_seconds,
+      retain_acked_messages: retain_acked_messages,
+      message_retention_duration: message_retention_duration,
+      labels: labels,
+      enable_message_ordering: enable_message_ordering,
+      filter: filter,
+      dead_letter_policy: dead_letter_policy,
+      retry_policy: retry_policy,
+      options: default_options
+    ]
+  end
```

Comment:
Oh this is so much nicer than all those horrible long lines that were impossible to read.

Reply:
I've always thought we should enable rubocop on the tests... But it would be quite a heavy lift at this point. I'll try to do more incremental improvement where I can.